data/json/effects.json

```diff
@@ -4191,5 +4191,15 @@
     "desc": [ "You are covered in glowing nether spores!" ],
     "apply_message": "The glowing spores settle on you!",
     "rating": "bad"
+  },
+  {
+    "id": "lightsnare",
+    "type": "effect_type",
+    "name": [ "Snared" ],
+    "desc": [ "For such a tiny creature, you're in a very big pickle." ],
+    "rating": "bad",
+    "show_intensity": false,
+    "//": "Don't multiply speed by 0 or else the effect will never be removed for creatures with 0 damage(e.g. rabbits)",
+    "enchantments": [ { "values": [ { "value": "SPEED", "multiply": 0.01 } ] } ]
```

Comment:
Enchantments doesn't work with monsters, use another way

Reply:
They do though? The rabbits in testing are all sitting perfectly still.

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/assets/84619419/c5b52e19-80ab-430d-9b88-9543eecabac2

Reply:
Maybe speed is an exception?
![IMG_20230514_193830](https://github.com/CleverRaven/Cataclysm-DDA/assets/67688115/a7907802-99a9-41ec-9d3b-af32a352e9fd)